### PR TITLE
Feat fix

### PR DESCRIPTION
**Pull request by Kothalkarpriya**
Drawer component added and fixed issues with styling and alignment for Different screen sizes

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -566,6 +583,69 @@ body {
...
     color: var(--hover-color);
 }
+.drawer{
+    margin:auto;
+}
+
+.drawer-container {
+    padding: 1rem;
+    background-color: var(--background-color);
+    width: 250px;
```
Comment (fahd23) [inline css/style.css]: If possible convert px to rem

File: css/style.css [hunk 2]
```diff
@@ -608,10 +689,12 @@ body {
     padding: 0.5rem 0;
     color: #ced4da;
```
Comment (fahd23) [inline css/style.css:687]: use CSS variables for consistency

**Review by psychoactiv — APPROVED** [review]
Code's clean , you can merge

**Review by fahd23 — APPROVED** [review]
Given some suggestions, Everything else looks good to me

File: css/style.css (hunk 1 above)
Comment (sushil-mod) [inline css/style.css]: if possible use rems

File: css/style.css (hunk 2 above)
Comment (sushil-mod) [inline css/style.css:690]: if possible use color variable

**Review by sushil-mod — APPROVED** [review]
just a few comments , then good to go

File: css/style.css (hunk 1 above)
Comment (Kothalkarpriya) [reply css/style.css]: I will change it

File: css/style.css (hunk 2 above)
Comment (Kothalkarpriya) [reply css/style.css:687]: Thanks, I forgot about this one

File: css/style.css (hunk 1 above)
Comment (Kothalkarpriya) [reply css/style.css]: I will change it

**Pull request merged**